Update processing and filtering information in docs to reflect scpca-nf release 0.1.3 (#46)
AlexsLemonade/scpca-docs: docs/sce_file_contents.md

The following are additional per-cell data columns included only in `filtered` objects. 
These metrics were calculated by using [`miQC`](https://bioconductor.org/packages/release/bioc/html/miQC.html), a package that jointly models proportion of reads belonging to mitochondrial genes and number of unique genes detected to predict low-quality cells. 

| Column name             | Contents                                                                                                                                                                                      |
| ----------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `prob_compromised`      | Probability that a cell is compromised (i.e., dead or damaged), as calculated by `miQC`                                                                                                       |

> | `miQC_pass`             | Indicates the likelihood of cells passing the default miQC filtering threshold of 0.75, `TRUE` is assigned to cells that have a low probability of being compromised (i.e. less than 0.75)    |

Reviewer: The miQC filtering is a bit more complex with options like `keep_all_below_boundary` which now defaults to true, so I think we should be a bit less concrete here.
```suggestion
| `miQC_pass`             | Indicates whether the cell passed the default miQC filtering. `TRUE` is assigned to cells that have a low probability of being compromised (i.e. `prob_compromised` less than 0.75) and those with sufficiently low mitochondrial content (see [miQC vignette](https://bioconductor.org/packages/release/bioc/vignettes/miQC/inst/doc/miQC.html#preventing-exclusion-of-low-mito-cells).)  |
```

> | `miQC_pass`             | Indicates whether the cell passed the default miQC filtering. `TRUE` is assigned to cells that have a low probability of being compromised, defined as cells with `prob_compromised` less than 0.75 and sufficiently low mitochondrial content (see [miQC vignette](https://bioconductor.org/packages/release/bioc/vignettes/miQC/inst/doc/miQC.html#preventing-exclusion-of-low-mito-cells).)  |

Reviewer: The current wording makes it seem like both low `prob_compromised` and low mito content are required, but it is more of an "and/or"... The recent changes mean that a cell can have a high `prob_compromised` as a result of the model fit stats, but if the mito content is low enough it is kept anyway. Trying to get that across succinctly is hard! I feel like the parenthetical may be the best way to do that if we want the numerical detail, or we can just leave it out?

I updated the second link to directly link from the text so we don't have two parentheticals.
```suggestion
| `miQC_pass`             | Indicates whether the cell passed the default miQC filtering. `TRUE` is assigned to cells with a low probability of being compromised (`prob_compromised` < 0.75) or [sufficiently low mitochondrial content](https://bioconductor.org/packages/release/bioc/vignettes/miQC/inst/doc/miQC.html#preventing-exclusion-of-low-mito-cells).  |
```

Author: Yes this makes sense now, thank you!
